### PR TITLE
feat(server/functions): GetPlayersFromCoords

### PR DESCRIPTION
**Pull request by Dark3581**
gets all players within an area

### CODE REVIEW

File: client/functions.lua [hunk 1]
```diff
@@ -10,6 +10,28 @@ function NDCore.Functions.GetCharacters()
     return NDCore.Characters
 end
 
+
+function NDCore.Functions.GetPlayersFromCoords(coords, distance)
+    local players = GetActivePlayers()
+    local ped = PlayerPedId()
```
Comment (Andyyy7666) [inline client/functions.lua:16]: ped variable is not used unless coords aren't there so it's better if it's moved below line 19

**Review by Andyyy7666 — CHANGES_REQUESTED** [review]

File: client/functions.lua (hunk 1 above)
Comment (Dark3581) [reply client/functions.lua]: ped var moved to where initialize

File: client/functions.lua [hunk 2]
```diff
@@ -10,6 +10,28 @@ function NDCore.Functions.GetCharacters()
...
+    if coords then
+        coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords
+    else
+        coords = GetEntityCoords(ped)
+    end
+    distance = distance or 5
+    local closePlayers = {}
+    for _, player in pairs(players) do
```
[the remark this answers is not in the document]
Comment (Dark3581) [reply client/functions.lua:23]: changed to ipairs

File: client/functions.lua [hunk 3]
```diff
@@ -10,6 +10,28 @@ function NDCore.Functions.GetCharacters()
     return NDCore.Characters
 end
 
+
+function NDCore.Functions.GetPlayersFromCoords(coords, distance)
```
[the remark this answers is not in the document]
Comment (Dark3581) [reply client/functions.lua]: changed to allow if coords is not defined

**Comment by Dark3581** [comment]
The requested changes should be done.

**Pull request merged**